docs/changelog.md

```diff
@@ -149,6 +149,11 @@ we clean up all configurations. This could lead to unexpected new log lines. Ple
 !!! note 
     Log4j is only used for standalone server deployments and JanusGraph testing.
 
+##### Add support for Java 11
+
+All tests are no running on Java 11. Java 11 won't yet anymore the full build distribution. 
```

Comment:
(nitpick) Should be _are no**w** running on Java 11_.

But I don't understand the second sentence at all. The full build distribution is removed now?